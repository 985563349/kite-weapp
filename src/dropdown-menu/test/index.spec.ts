import path from 'node:path';
import simulate from 'miniprogram-simulate';

describe('dropdown-menu', () => {
  const KDropdownMenu = simulate.load(path.resolve(__dirname, '../dropdown-menu'), 'k-dropdown-menu', {
    rootPath: path.resolve(__dirname, '../../'),
  });

  test('should render dropdown-menu and match snapshot', () => {
    const comp = simulate.render(KDropdownMenu);
    comp.attach(document.createElement('parent-wrapper'));

    expect(comp.toJSON()).toMatchSnapshot();
  });
});
