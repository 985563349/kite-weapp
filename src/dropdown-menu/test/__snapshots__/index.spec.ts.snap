// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dropdown-menu should render dropdown-menu and match snapshot 1`] = `
<wx-view
  class="k-dropdown-menu k-class"
  id=""
  style=""
  catch:touchmove="noop"
/>
`;
